### PR TITLE
speedup ci using verify_at_rows

### LINKED ISSUE #701: ci: speedup ci of tx circuit using verify_at_rows
### Describe the feature you would like

after https://github.com/scroll-tech/zkevm-circuits/pull/669, degree changed to 20 in ecc / sig / tx circuit. CI times increase from 20 minutes to 30 minutes. Need to fix that.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lispc**
resolves https://github.com/scroll-tech/zkevm-circuits/issues/701

### CODE REVIEW

File: zkevm-circuits/src/tx_circuit/test.rs [hunk 1]
```diff
@@ -114,10 +114,9 @@ fn run<F: Field>(
     max_txs: usize,
     max_calldata: usize,
 ) -> Result<(), Vec<VerifyFailure>> {
-    let k = max(
-        20,
-        log2_ceil(TxCircuit::<F>::min_num_rows(max_txs, max_calldata)),
-    );
+    let active_row_num = TxCircuit::<F>::min_num_rows(max_txs, max_calldata);
+
+    let k = max(20, log2_ceil(active_row_num));
```
Comment (kunxian-xia) [inline zkevm-circuits/src/tx_circuit/test.rs:119]: k = `min(20, log2_ceil(active_row_num)` ?

Comment (lispc) [reply zkevm-circuits/src/tx_circuit/test.rs:119]: sig circuit (embedded inside tx circuit) use LOG_NUM_ROWS=20, so at least need 20?

**Comment by lispc** [comment]
oh no it is super circuit with degree 20 that is slow.... 

i will try switch back to 19 in this repo..

File: zkevm-circuits/src/tx_circuit/test.rs (hunk 1 above)
Comment (kunxian-xia) [reply zkevm-circuits/src/tx_circuit/test.rs:119]: yeah, you're right.

**Pull request merged**